hazelcast/src/main/java/com/hazelcast/internal/cluster/impl/ConfigCheck.java

```diff
@@ -208,7 +202,7 @@ public void writeData(ObjectDataOutput out) throws IOException {
     @Override
     public void readData(ObjectDataInput in) throws IOException {
         groupName = in.readUTF();
-        groupPassword = in.readUTF();
+        in.readUTF();
```

Comment:
You're not reading group-password but it's still written to the stream and broadcasted. So, it'll be visible in multicast packets.